microbench/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+if(MSVC)
+    return()
+endif()
+
+set(CMAKE_REQUIRED_LIBRARIES rt)
```

Comment:
`CMAKE_REQUIRED_LIBRARIES` may have value, append insted of set.

Reply:
`CMAKE_REQUIRED_LIBRARIES ` isn't set in the top CMakeLists.txt so I guess it's always empty here. Do you agree?

Reply:
Could it be set from command line? It's better to make it 100% correct from start.

Reply:
I have double checked what CMAKE_REQUIRED_LIBRARIES is needed for. It is not a public CMake variable but it is used as an input to check_library_exists() or check_symbol_exists() functions. In that sense it is a locally scoped variable for a particular call of those two functions. Therefore it's even mandatory to reset these variable to a new value before you want to run a test in order to avoid side effects from previous tests.

Reply:
ANY CMake variable can be set from command line.

What if math library is not m? Somebody can set `CMAKE_REQUIRED_LIBRARIES` and `CMAKE_C_STANDARD_LIBRARIES` to get it work.

Is it impossible? Nope.

```m4
*-ncr-sysv4.3*)
  AC_CHECK_LIB(mw, _mwvalidcheckl, LIBM="-lmw")
*)
```

Reply:
As I said this variable has to be reset each time you test something because if you set it to foo while there is no foo on your system then the testing foo will fail. Supposing that the test is optional all the other tests will falsely fail also (including some mandatory).

Reply:
Hm, you're right. Thanks